android/game/build.gradle

```diff
@@ -0,0 +1,66 @@
+plugins {
+    id 'com.android.library'
+    id 'maven-publish'
+}
+def gameHubVersion = '1.0.0'
+def gameHubArtifact = 'alpha-3'
+def gameHubArtifactDebug = 'alpha-3-debug'
+
+android {
+    compileSdk 30
+
+    defaultConfig {
+        minSdk 16
+        targetSdk 29
+        versionCode 1
+        versionName "1.0"
```

Comment:
Use variables which are defined above and remove all these numbers and use variables, also as a suggestion use separated file and import in this file